packages/graphiql-react/src/editor/hooks.ts

```diff
@@ -459,7 +458,7 @@ export function useOptimisticState([
       lastStateRef.current.pending === null &&
       lastStateRef.current.last !== newState
     ) {
-      // No pending updates and change has occurred... send it upstream
+      // No pending updates and change have occurred... send it upstream
```

Comment:
The active subject is the singular `change`, and the verb should match.
```suggestion
      // No pending updates and change has occurred... send it upstream
```